moveit_experimental/moveit_jog_arm/config/ur_simulated_config.yaml

```diff
@@ -47,7 +47,7 @@ joint_limit_margin: 0.1 # added as a buffer to joint limits [radians]. If moving
 cartesian_command_in_topic: jog_server/delta_jog_cmds  # Topic for incoming Cartesian twist commands
 joint_command_in_topic: jog_server/joint_delta_jog_cmds # Topic for incoming joint angle commands
 joint_topic:  joint_states
-warning_topic: jog_server/halted # Publish boolean warnings to this topic
+status_topic: jog_server/status # Publish string status messages to this topic
```

Comment:
The idea is to publish status messages in general but all the messages end up being warnings. I think the naming here is good but for example `WarningCodeEnum` could be just `StatusCode` to make it more expandable in the future. Maybe you want to add info messages or debug or error messages at some point?

Reply:
ok, renaming WarningCodeEnum->StatusCode sounds fine to me. Done